packages/use-intl/src/react/IntlContext.tsx

```diff
@@ -21,4 +21,7 @@ export type IntlContextShape = {
 
 const IntlContext = createContext<IntlContextShape | undefined>(undefined);
 
+// used for getTranslation helper
+export const IntlContextValue: IntlContextShape = {} as any;
```

Comment:
This is problematic as it breaks the isolated nature of context and instead creates a global singleton.